.changesets/feat_geal_plan_cache_warmup_doc.md

```diff
@@ -0,0 +1,13 @@
+### Query plan cache warm-up improvements ([Issue #3704](https://github.com/apollographql/router/issues/3704))
+
+The `warm_up_queries` option enables quicker schema updates by precomputing query plans for your most used cached queries and your persisted queries. When a new schema is loaded, a precomputed query plan for it may already be in in-memory cache.
+
+We made a series of improvements to this feature to make it more usable:
+* It is now active by default and warms up the cache with the 30% most used queries of the previous cache. The amount is still configurable, and it can be deactivated by setting it to 0.
+* We added new metrics to track the time spent loading a new schema and planning queries in the warm-up phase. You can also measure the query plan cache usage, to know how many entries are used, and the cache hit rate, both for the in memory cache and the distributed cache.
```

Comment:
@Geal is this part of the work Benjamin did [in this changeset](https://github.com/apollographql/router/pull/3861/files#diff-0a6b9559d47c40fd09b21288e986861d161480485228474398369dc2535257e6), or is that something entirely separate?

Reply:
the work of Benjamin is part of this changeset. #3704 is an umbrella issue for the query planner improvements. So the changesets should be merged

Reply:
ty!